scss/components/_button.scss

```diff
@@ -0,0 +1,76 @@
+@use '../abstracts/variables' as *;
+
+button,
+.styled-button {
+  align-items: center;
+  background-color: #{$neutral--80};
+  border: 0.5px solid #{$neutral--80};
+  border-radius: 3.625rem;
+  color: #{$neutral--00};
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  padding: 0.84375rem 0;
+  font-size: 0.6875rem;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 1rem;
+  transition: background-color 0.25s ease, height 0.25s ease;
+  min-width: 6.45rem;
+
+  .label {
+    cursor: pointer;
+    pointer-events: inherit;
+  }
+
+  &--small {
+    padding: 0.5rem 1rem;
+    min-width: unset;
+  }
+
+  &--large {
```

Comment:
I assume these are just examples? since when looking at large and medium btn they look the same, and there is no --medium modifier here?

Reply:
yup yup, i left this here for now, our "medium" style is actually an icon and haven't implemented that yet